### PR TITLE
feature(yaml): replace yaml parser library to support multi new line

### PR DESCRIPTION
**Pull request by RyuNen344**
YamlSwift can't parse yaml like below

```yaml
manual:
  - source: https://github.com/firebase/firebase-ios-sdk
    name: Firebase
    version: 4.0.0
    body: |2
                                      Apache License
                                Version 2.0, January 2004
                              http://www.apache.org/licenses/
        TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
```

so I replaced yaml parser Library, YamlSwift to Yams

### CODE REVIEW

File: Tests/LicensePlistTests/Entity/ConfigTests.swift [hunk 1]
```diff
@@ -17,6 +17,10 @@ class ConfigTests: XCTestCase {
                                                source: "https://webrtc.googlesource.com/src",
                                                nameSpecified: "Web RTC",
                                                version: "M61"),
+                                        Manual(name: "Firebase",
```
Comment (RyuNen344) [inline Tests/LicensePlistTests/Entity/ConfigTests.swift:20]: This test fails until this PR merge

File: Tests/LicensePlistTests/Entity/SwiftPackageManagerTests.swift [hunk 2]
```diff
@@ -187,12 +187,11 @@ class SwiftPackageManagerTests: XCTestCase {
                                                   version: "5.3.0",
                                                   packageDefinitionVersion: 2))
         let packageLast = try XCTUnwrap(packages.last)
-        XCTAssertEqual(packageLast, SwiftPackage(package: "yamlswift",
-                                                 repositoryURL: "https://github.com/behrang/YamlSwift.git",
-                                                 revision: "287f5cab7da0d92eb947b5fd8151b203ae04a9a3",
-                                                 version: "3.4.4",
+        XCTAssertEqual(packageLast, SwiftPackage(package: "yams",
```
Comment (RyuNen344) [inline Tests/LicensePlistTests/Entity/SwiftPackageManagerTests.swift:190]: Also, this test fails until this PR merge

**Comment by RyuNen344** [comment]
Thanks for allowing to run GitHub Actions, I will fix lint indicates

**Comment by RyuNen344** [comment]
I fixed issues.

**Comment by RyuNen344** [comment]
```
/home/runner/work/LicensePlist/LicensePlist/Sources/LicensePlistCore/Entity/Manual.swift:45:1: error: Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)
/home/runner/work/LicensePlist/LicensePlist/Sources/LicensePlistCore/Entity/Manual.swift:50:1: error: Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)
/home/runner/work/LicensePlist/LicensePlist/Sources/LicensePlistCore/Entity/Manual.swift:55:1: error: Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)
```

sorry, I will refix..... 😢

**Comment by RyuNen344** [comment]
I fixed lint error, Could you approve to run GitHub Actions?

**Comment by mono0926** [comment]
@RyuNen344 

Thanks, released 👍 
https://github.com/mono0926/LicensePlist/releases/tag/3.24.10

**Pull request merged**